components/Learn/LearnStartLearning.vue

```diff
@@ -181,8 +181,8 @@ const learningSections: LearningSection[] = [
         description:
           "This free course covers quantum information at a detailed mathematical level. Join John Watrous as you explore quantum information, quantum algorithms, and how to understand and mitigate noise.",
         cta: {
-          label: "Go to this course",
-          url: "/learn/course/basics-quantum-information",
+          label: "Take this course in IBM Quantum Learning",
+          url: "https://qiskit.org/learn/course/basics-quantum-information",
```

Comment:
Will this link redirect to the new IBM Quantum Learning site? (Same goes for line 198)